app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/EditServiceRequest.php

```diff
@@ -68,9 +71,10 @@ public function form(Form $form): Form
                 Select::make('status_id')
                     ->relationship('status', 'name')
                     ->label('Status')
-                    ->options(fn () => ServiceRequestStatus::optionsByClassification())
+                    ->options(fn () => ServiceRequestStatus::optionsByClassification(true))
```

Comment:
I like to use named parameters when the values aren't immediately obvious what you're passing in. What do you think?

Reply:
I ended up removing the function.